corehq/ex-submodules/toggle/shortcuts.py

```diff
@@ -72,23 +72,18 @@ def parse_toggle(entry):
     return namespace, entry
 
 
-def find_with_toggle_enabled(toggle, namespace):
-    from corehq.toggles import ALL_NAMESPACES
-
-    if namespace not in ALL_NAMESPACES:
-        raise ValueError('Unknown toggle namespace "{}"'.format(namespace))
+def find_users_with_toggle_enabled(toggle):
     try:
         doc = Toggle.get(toggle.slug)
     except ResourceNotFound:
         return []
-    return [user[len(namespace) + 1:] for user in doc.enabled_users if user.startswith(namespace)]
-
-
-def find_users_with_toggle_enabled(toggle):
-    from corehq.toggles import NAMESPACE_USER
-    return find_with_toggle_enabled(toggle, namespace=NAMESPACE_USER)
+    return filter(lambda user: ':' not in user, doc.enabled_users)
```

Comment:
Is it possible that a username could contain `:`? I think it would be safer to do something like
```py
prefixes = tuple(ns + ":" for ns in ALL_NAMESPACES if ns is not None)
return [u for u in doc.enabled_users if not u.startswith(prefixes)]
```
Any username that starts with `domain:` is doomed (is that possible?), but there have always been issues like that because of how the namespacing was implemented way back when.

Reply:
Actually, just tested this and it appears we do not allow usernames containing `:` so maybe it doesn't matter?

Reply:
Yeah, but I didn't know `startswith` can take a tuple. So now I have to change it. :grin: